career_development/ladder.md

```diff
@@ -0,0 +1,114 @@
+# Abtion Dev Career Ladder
+
+## High Level Company Values
+1. Bonded by Love for Product
+1. Continuous Delivery
+1. Customer Comes First
+1. Heavily Team Oriented
+1. High Employee Retention
+1. High Quality Code Base
+1. Open Communication 
+1. Pair-Programs
+
+## Dev Values : Simple, Test-Driven, Iterative, Communicative, Respectful, Skilled 
+
+### Level 1
+
+#### simple 
+-[ ] is working on naming variables and learning to extract methods
```

Comment:
Depending on the markdown interpreter, we might need a space between the `-` and `[ ]`. I'm not sure how Github will interpret this.